src/editions/transitioning.md

```diff
@@ -1,47 +1,30 @@
 # Transitioning your code to a new edition
 
-Transitioning between editions is built around lints. Fundamentally, the
-process works like this:
+New editions might change the way you write Rust -- they add syntax, language,
+and library features but also remove others. For example, `async`/`await` will
+be available in Rust 2018, but not Rust 2015. Despite this it's our intention
```

Comment:
Saying that async await will be available in Rust 2018 is true; but not at the release date this year, which a user may reasonably construe from this wording without any info as to the contrary.